src/utils/trust.details.ts

```diff
@@ -61,24 +64,33 @@ const getPageProperties = (
       beneficialOwners: boAvailableForTrust,
     },
     formData,
+    isReview,
     errors,
     url: getUrl(isUpdate),
   };
 };
 
-export const getTrustDetails = (req: Request, res: Response, next: NextFunction, isUpdate: boolean): void => {
+export const getTrustDetails = (req: Request, res: Response, next: NextFunction, isUpdate: boolean, isReview: boolean): void => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
 
-    const trustId = req.params[config.ROUTE_PARAM_TRUST_ID];
+    let trustId;
+    if (isReview) {
+      const trustInReview = appData.update?.review_trusts?.find(trust => trust.review_status);
```

Comment:
could you use the review_trusts util method here? `getTrustInReview`

Reply:
Tried using it in place but doesn't seem to let the mock data read through. Believe ### getTrustInReview is a different retrieval

Reply:
@corybudd-ch don't worry about this - I'm creating a PR here to update the model so this is clearer.

Reply:
to clarify - the review the trust page will have a flag like the rest, review_trust_details, and in_review will be set to true again when the orchestrator sets up the review_status object.

Reply:
sweet, leave as is then